packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -25,75 +22,48 @@ import type {
 } from "../util/index.js";
 import { validateUsageError } from "./utils.js";
 import { takeJsonSnapshot, useSnapshotDirectory } from "./snapshots/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { describeHydration } from "./simple-tree/utils.js";
 
 const schemaFactory = new SchemaFactoryAlpha("test");
```

Comment:
Note for reviewers: I definitely recommend enabling the "hide whitespace" option before reviewing these changes. There's a fair amount of whitespace only changes.